server/controllers/profile.controller.js

```diff
@@ -41,7 +48,10 @@ class ProfileController {
       username,
       bio,
       image,
-      articles: `${userActivity[0]} articles written.`,
+      articlesWritten: `${userActivity[0].count} articles written.`,
+      articlesWrittenList: userActivity[0].rows,
+      articlesRead: `You have read ${userActivity[1].count} article(s).`,
+      articlesReadList: userActivity[1].rows,
       // following: `You are following ${userActivity[1]} authors.`,
```

Comment:
The code base would be neater if comments can be removed

Reply:
Right. I need the comments for now though, as the feature for following and following has not been merged yet. I might forget to add it if the comments are not there

Reply:
Ok.

Reply:
Comments removed.